### DISCUSSION TITLE
UOS虚拟机在使用QXl显卡时，自适应失败

### DISCUSSION THREAD

**Discussion by zkm111** [linuxdeepin/developer-center#7692] [Q&A | 问答板块]
- 内核：4.19.0-amd64-desktop
- Qemu：6.2.0
- 显卡信息
![B4DBEEDE-C5BE-4e45-A278-2204F64D9AE8](https://github.com/linuxdeepin/developer-center/assets/47200664/869e4159-18db-4e39-a9c3-5954be83f091)
![9380CB8B-449D-4b6f-9BF2-A57BA9491CFC](https://github.com/linuxdeepin/developer-center/assets/47200664/0e79559a-2f2f-4fc3-be33-fb6f5897c1dd)

- 系统日志
![image](https://github.com/linuxdeepin/developer-center/assets/47200664/620a9245-53d3-4b25-92c7-4b0ece7d4a07)

目前已更换qxl.ko模块进行调试，确定qxl驱动有正常收到[QXL_INTERRUPT_CLIENT_MONITORS_CONFIG](https://elixir.bootlin.com/linux/v4.19/C/ident/QXL_INTERRUPT_CLIENT_MONITORS_CONFIG)。但https://github.com/linuxdeepin/startdde/blob/master/display/xorg.go#L512 中会有报错。不确定这个是否会影响屏幕分辨率修改。还有如何确定daemon/display对应版本。因为目前系统内报错的行数和master对应不上。如何开启daemon/display对应debug。

**Comment by chuxinyuan** [comment]


**Reply by zkm111** [reply]
你也有遇到这个情况么
